index.js

```diff
@@ -46,7 +47,18 @@ async function run() {
     await exec.exec(`${pkgManager} run build ${buildArgs}`, []);
     console.log("Finished building your site.");
 
-    await exec.exec(`${pkgManager} run scully -- --nw ${scullyArgs}`, []);
+    // determine the scully version
+    const packageJsonRaw = await fs.readFile('./package.json', 'utf8');
+    const packageJsonParsed = JSON.parse(packageJsonRaw);
+    const scullyVersion = packageJsonParsed.dependencies['@scullyio/scully']
+      .replace('^', '')
+      .replace('~', '');
+    // add the `--nw` flag if scully version is below or equal `0.0.85`
+    if (scullyVersion <= '0.0.85') {
```

Comment:
How would you feel about using node-semver for this version comparison instead?

I'm just fuzzy on the possible edge cases that using raw string comparison would have.

Reply:
sure, makes much more sense